charts/gardener-extension-shoot-oidc-service/templates/service.yaml

```diff
@@ -9,14 +9,16 @@ metadata:
   name: gardener-extension-{{ .Values.serviceName }}
   namespace: {{ .Release.Namespace }}
   annotations:
-    networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
-    networking.resources.gardener.cloud/from-policy-pod-label-selector: all-seed-scrape-targets
+    networking.resources.gardener.cloud/from-all-seed-scrape-targets-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
     networking.resources.gardener.cloud/from-policy-allowed-ports: '[{"port":{{ .Values.metrics.port }},"protocol":"TCP"}]'
     networking.resources.gardener.cloud/namespace-selectors: '[{"matchLabels":{"kubernetes.io/metadata.name":"garden"}}]'
     networking.resources.gardener.cloud/pod-label-selector-namespace-alias: extensions
 {{- if .Values.ignoreResources }}
     resources.gardener.cloud/ignore: "true"
 {{- end }}
+    # TODO: This label approach is deprecated and no longer needed in the future. Remove them as soon as gardener/gardener@v1.75 has been released.
+    networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.webhookConfig.serverPort }}}]'
```

Comment:
Maybe I am missing something, but isn't this label still needed (the server port)? I do not see it removed in other PRs. Maybe you meant to change the labels regarding the `metrics` port? See [this](https://github.com/gardener/gardener/commit/b14706780cb74902404aa44351bd39124aee48ae#diff-0f942ac1144b309bca6a3b86b204f6a3e3daa1877a9651e1c3d291f121b72927L8-L9) as an example.